aframe/src/component-anchor-nft.js

```diff
@@ -178,6 +178,12 @@ AFRAME.registerComponent('arjs-anchor', {
         arWorldRoot.updateMatrixWorld(true)
         arWorldRoot.matrixWorld.decompose(this.el.object3D.position, this.el.object3D.quaternion, this.el.object3D.scale)
 
+        document.addEventListener('arjs-nft-init-data', function(ev) {
+            var msg = ev.detail;
+            this.el.object3D.position.z = -(msg.height / msg.dpi * 2.54 * 10)/2.0;
```

Comment:
Sorry my understanding of aframe and compotents is limited but should it be this.el.object3D.position.setZ? 
 
 
 https://threejs.org/docs/#api/en/math/Vector3.setZ

Reply:
i don't remember exactly what was the problem,but i think the event is not listened inside the registerComponent. I had to check again.

Reply:
i will close this PR, because the old code. In regards at the problem maybe it's possible to do the same as for the three.js example? i will see when i will open the new PR.